examples/src/node-to-roc.c

```diff
@@ -7,6 +7,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <stdint.h>
```

Comment:
I'm not sure why it's showing this as a diff since it's [unchanged from the target branch](https://github.com/vendrinc/roc-esbuild/blob/7f7cff46e360e8f6018a0aa6f8c635d7706aa359/src/node-to-roc.c#L10). 🤷